Mage/src/main/java/mage/game/command/emblems/EmblemOfCard.java

```diff
@@ -0,0 +1,65 @@
+package mage.game.command.emblems;
+
+import mage.MageObject;
+import mage.abilities.AbilityImpl;
+import mage.cards.Card;
+import mage.cards.decks.DeckCardInfo;
+import mage.cards.decks.importer.CardLookup;
+import mage.cards.mock.MockCard;
+import mage.constants.Zone;
+import mage.game.command.Emblem;
+import org.apache.log4j.Logger;
+
+import java.util.stream.Collectors;
+
+public final class EmblemOfCard extends Emblem {
+    private static final Logger logger = Logger.getLogger(EmblemOfCard.class);
+    public static Card cardFromDeckInfo(DeckCardInfo info) {
+        return CardLookup.instance.lookupCardInfo(info.getCardName(), info.getSetCode(), info.getCardNum())
+                .orElseThrow(() -> new IllegalArgumentException("No real card for DeckCardInfo" + info.getCardName()))
+                .getCard();
+    }
+    public EmblemOfCard(Card card, Zone zone) {
+        super(card.getName());
+        if (card instanceof MockCard) {
+            MockCard mock = (MockCard)card;
+            card = CardLookup.instance.lookupCardInfo(card.getName(), card.getExpansionSetCode(), card.getCardNumber())
+                    .orElseThrow(() -> new IllegalArgumentException("No real card for mock card " + mock.getFullName(true)))
+                    .getCard();
+        }
+        this.getAbilities().addAll(card.getAbilities().stream().filter(
+                ability -> zone.match(ability.getZone())
+        ).map(ability -> {
+            if (ability instanceof AbilityImpl && ability.getZone() == zone) {
+                return ((AbilityImpl)ability).copyWithZone(Zone.COMMAND);
```

Comment:
- The entire client is marked as "beta", and regularly has bugs.
- I'll admit I thought this implementation would have problems, but as mentioned above, I've tested every perverse card I could think of and never had anything worse than "the emblem doesn't do anything". Find a single example of instability and I'll agree that a warning makes sense, but as far as I can tell, abilities are already written to handle "my source isn't a permanent on the battlefield" as a natural product of how Magic works. (Meanwhile, multiple clash cards were bugged for months in a way that broke all active games on the entire server; they weren't even temporarily removed or disabled, just left in that broken state for many releases.)
- In what way is it "unsupported by MTG rules"? Abilities printed on emblems are worded exactly the same as abilities on permanents, but function from the command zone rather than the battlefield. CR 114.4: `Abilities of emblems function in the command zone.` 
  - While this feature doesn't correspond to any variant listed in the rules, it's directly based on an Arena feature used by WotC to create events. Note that Alchemy mechanics like conjure are similarly not present in the CR, yet included in XMage.

Reply:
Decided "some cards don't do anything" is maybe enough reason for a warning. How's this?
![updated emblem cards section](https://github.com/magefree/mage/assets/19716820/7d35c986-9fbf-49f3-9d01-4ef20db87e9a)

Reply:
Also decided Planechase has more than enough oddities to need a warning/description.
![planechase description](https://github.com/magefree/mage/assets/19716820/18c4015d-8290-46c5-92cf-8e2c0683eabf)